### PR TITLE
[RES-988] Python sample app for Nitrous.io

### PR DESCRIPTION
**Pull request by mariusrusu**
As a nitrous.io user I need to have a Sendgrid python sample app in order to see how to use sendgrid to send emails.
- [x] CR'd
- [x] QA'd
- [x] Dependencies satisfied
- [x] Ready to deploy

### CODE REVIEW

File: SGapp/settings.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.6/howto/static-files/
+
+STATIC_URL = '/SGapp/static/'
+
+
```
Comment (laur-craciun) [inline SGapp/settings.py]: add a comment here, so everyone will know that this constants need to be updated

File: SGapp/views.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+from SGapp.form import MailForm
+import sendgrid
+
+def index(request):
+    if request.method == 'POST':
+
+        form = MailForm(request.POST)
+        if form.is_valid():
+
```
Comment (laur-craciun) [inline SGapp/views.py]: delete empty line

File: README.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+
+    SENDGRID_USERNAME = '<sendgrid_username>';
+    SENDGRID_PASSWORD = '<sendgrid_password>';
+
+Start python server
+
+    python manage.py runserver
+    # or
+    python manage.py runserver 0.0.0.0:8000
+
```
Comment (laur-craciun) [inline README.md:31]: please add here, that the user can now open the app in his browser and a print-screen with the sample app in browser

File: README.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
-sendgrid-python-django-sample-app
+SendGrid Python Django Sample App
 =================================
+
+This git repository helps you to send emails quickly and easily through SendGrid using Python and Django.
+
+Create an SendGrid account at http://sendgrid.com/pricing.html
```
Comment (laur-craciun) [inline README.md]: please use this link to pricing page: http://sendgrid.com/transactional-email/pricing

Comment (mariusrusu) [inline README.md]: done

File: SGapp/settings.py (hunk 1 above)
Comment (mariusrusu) [inline SGapp/settings.py]: done

File: SGapp/views.py (hunk 2 above)
Comment (mariusrusu) [inline SGapp/views.py]: done

File: README.md (hunk 3 above)
Comment (mariusrusu) [inline README.md:31]: done

**Pull request merged**